packages/desktop/electron/main.js

```diff
@@ -179,34 +179,11 @@ if (app.isPackaged) {
  * Check URL against allowlist
  */
 function isUrlAllowed(targetUrl) {
-    const externalAllowlist = [
-        // IOTA Foundation
-        'blog.iota.org',
-        'chrysalis.iota.org',
-        'chrysalis.docs.iota.org',
-        'discord.iota.org',
-        'firefly.iota.org',
-        'iota.org',
-        'privacy@iota.org',
-        'wiki.iota.org',
-        'explorer.iota.org',
-
-        // Assembly / Shimmer
-        'assembly.sc',
-        'shimmer.network',
-        'explorer.shimmer.network',
-
-        // GitHub
-        'github.com/iotaledger/firefly/issues',
-        'github.com/iotaledger/firefly/issues/new/choose',
-
-        // Other
-        'support.ledger.com',
-    ]
+    const externalBlacklist = []
```

Comment:
Why would the blacklist be in electron still? Would this not be shared for desktop and mobile? Can add a new task and PR because we probably want to show a warning that its blocked.